### ISSUE TITLE
Prepare next release

### ISSUE THREAD

**Issue by julien-truffaut** [optics-dev/Monocle#1325]
Here is the list of changes since 3.1.0: https://github.com/optics-dev/Monocle/compare/v3.1.0...master

We bumped the major version of the scala compiler to 3.2.1. I am not sure what is the impact. 
I think the next version will be 3.2.0.

**Comment by armanbilge** [comment]
> I think the next version will be 3.2.0.

Yes indeed, I already bumped the "base version" in the `build.sbt` to reflect that.

https://github.com/optics-dev/Monocle/blob/9cfdb174cc55be3d8953243c192c141362cb9f38/build.sbt#L5

https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway

**Comment by armanbilge** [comment]
I think before the next release, the `PGP_SECRET` needs to be updated. I believe it has expired based on the CI logs.

You can generate a new key here:
https://typelevel.org/sbt-typelevel/secrets.html#pgp-key

If you use the online key generator, make sure to delete the `PGP_PASSPHRASE` secret.

**Comment by julien-truffaut** [comment]
Thanks, I will regenerate one.

**Comment by yuriy-yarosh** [comment]
@armanbilge scala-native release would be highly appreciated as well.

**Comment by armanbilge** [comment]
@yuriy-yarosh indeed, fortunately there is a snapshot available while we are waiting :)
https://s01.oss.sonatype.org/content/repositories/snapshots/dev/optics/monocle-core_native0.4_3/3.2-dcad42c-SNAPSHOT/

**Comment by julien-truffaut** [comment]
Sorry for the delay. I will try to do it next week.

**Comment by julien-truffaut** [comment]
@armanbilge I updated PGP_SECRET and deleted PGP_PASSPHRASE. What is the next step? should I create and publish a tag?

**Comment by armanbilge** [comment]
@julien-truffaut great! Yes, you can either create/push a tag locally, or you can just create the release from the GH UI.
https://github.com/optics-dev/Monocle/releases/new

**Comment by julien-truffaut** [comment]
Thanks, I didn't know about this UI. I triggered the build, let's cross our fingers :)

**Comment by julien-truffaut** [comment]
publication failed with this error:
```
2023-01-03 14:39:25.982Z error [SonatypeClient]     Failed: signature-staging, failureMessage:No public key: Key with id: (48fbfd133fedb341) was not able to be located on <a href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>. Upload your public key and try the operation again.  - (SonatypeClient.scala:385)
2023-01-03 14:39:25.984Z error [Sonatype] [STAGE_FAILURE] Failed to close the repository.  - (Sonatype.scala:440)
```

**Comment by armanbilge** [comment]
@julien-truffaut did you submit the public key to the key server as explained here?
https://typelevel.org/sbt-typelevel/secrets.html#pgp-key

**Comment by julien-truffaut** [comment]
Yes but I must have made a mistake. I regenerated a key and restarted the build and now it works :) Thanks

**Comment by armanbilge** [comment]
Artifacts are live:
https://repo1.maven.org/maven2/dev/optics/monocle-core_3/3.2.0/

**Comment by julien-truffaut** [comment]
I will try to write the release notes over the next week or two.

**Comment by armanbilge** [comment]
There's also an "auto-generate" release notes button in the GH UI :)

**Comment by julien-truffaut** [comment]
Yeah but it isn't great

**Comment by julien-truffaut** [comment]
@armanbilge I updated the release notes. If you have a minute wold you mind reviewing it?

**Comment by armanbilge** [comment]
@julien-truffaut LGTM 👍 thanks!
